src/localization/components/LocalePicker.js

```diff
@@ -41,18 +53,31 @@ const LocalePicker = () => {
   }
 
   const currentLocale = i18n.resolvedLanguage;
+  const selectProps = {
+    size: 'small',
+    value: currentLocale,
+    onChange: handleChange,
+    inputProps: {
+      'aria-label': t('localization.locale_select_label', {
+        name: t(getLocaleLabel(currentLocale)),
+      }),
+    },
+  };
+
+  if (variant === 'white') {
+    return (
+      <WhiteSelect {...selectProps}>
+        {Object.keys(LOCALES).map(locale => (
+          <MenuItem key={locale} value={locale}>
+            {t(getLocaleLabel(locale))}
+          </MenuItem>
+        ))}
+      </WhiteSelect>
```

Comment:
issue: this is pretty duplicative. Can we have one select component with a variant parameter?